src/circuits/joinsplit.tcc

```diff
@@ -266,15 +271,21 @@ public:
                 unpacked_inputs[i].insert(
                     unpacked_inputs[i].end(),
                     output_commitments[j]->bits.rbegin() +
-                        (HashT::get_digest_len() - FieldT::capacity()),
+                        safe_subtraction(
+                            HashT::get_digest_len(),
+                            FieldT::capacity()
+                        ),
                     output_commitments[j]->bits.rend());
             }
 
             // Initialize the unpacked input corresponding to the h_sig
             unpacked_inputs[NumOutputs + NumInputs].insert(
                 unpacked_inputs[NumOutputs + NumInputs].end(),
                 h_sig->bits.rbegin() +
-                    (HashT::get_digest_len() - FieldT::capacity()),
+                        safe_subtraction(
```

Comment:
Looks like it's indented further than the old code...
I've just notived that I missed the code formatting checks in the github actions, hence the CI doesn't pick this up ... fixing that now.

Reply:
Reinstated formatting checks in #172 
Sorry about that.

Reply:
Thanks @dtebbs will wait for the checks to pass on #172 and merge it. Will fix the formatting issues afterwards

Reply:
Thanks for merging #172.  Hopefully th CI will show errors once this is rebased onto `develop`, but if not let me know and I'll invesstigate.